UPGRADE.md

```diff
@@ -1,5 +1,13 @@
 # Upgrade to 3.0
 
+## BC Break: Removed the `doctrine` binary.
+
+The documentation explains how the console tools can be bootstrapped for
```

Comment:
A link to the mentioned documentation will help here.

Reply:
Done.